plugins/plugin-java/che-plugin-java-ext-lang-server/src/main/java/org/eclipse/che/plugin/java/server/rest/RefactoringService.java

```diff
@@ -78,6 +81,11 @@ public RefactoringService(RefactoringManager manager) {
   @Produces("text/plain")
   public String createMoveRefactoring(CreateMoveRefactoring cmr)
       throws JavaModelException, RefactoringException {
+    LOG.info(
+        "createMoveRefactoring started in "
+            + System.currentTimeMillis()
+            + " ms with "
+            + cmr.toString());
```

Comment:
Use please LOG.info("createMoveRefactoring started in  {} " ms with " {}", System.currentTimeMillis(), cmr.toString())